examples/pony/alphabet/data_gen/Makefile

```diff
@@ -0,0 +1,37 @@
+# include root makefile
+ifndef ROOT_MAKEFILE_MK
+include ../../../Makefile
+endif
+
+# prevent rules from being evaluated/included multiple times
+ifndef $(abspath $(lastword $(MAKEFILE_LIST)))_MK
+$(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
+
+# uncomment to disable generate test related targets in this directory
+TEST_TARGET := false
+
+# uncomment to disable generate pony related targets (build/test/clean) for pony sources in this directory
+#PONY_TARGET := false
+
+# uncomment to disable generate exs related targets (build/test/clean) for elixir sources in this directory
+EXS_TARGET := false
+
+# uncomment to disable generate docker related targets (build/push) for Dockerfile in this directory
+DOCKER_TARGET := false
+
+# uncomment to disable generate recursing into Makefiles of subdirectories
+#RECURSE_SUBMAKEFILES := false
+
+
+ALPHABET_PONY_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+
+# standard rules generation makefile
+include $(rules_mk_path)
+
+test-examples-pony-alphabet: alphabet_pony_test
+
+alphabet_pony_test:
+	echo TODO, this is a placeholder.
```

Comment:
You can remove lines 26 and 31 - 34 as they're no longer required because there's no custom test target needed.

Also, as a general note, `make` targets and variables need to be unique across the entire repository (or else `make` prints warning and might not behave as expected). As a result, `alphabet_pony_test` shouldn't be reused in this `Makefile` as it's already defined in `examples/pony/alphabet/Makefile`. Same with `test-examples-pony-alphabet` and `ALPHABET_PONY_PATH`.

Reply:
This change is outside of the scope that I expected for this issue. I assumed that we would be replacing calls to `stable` in situations where there was an existing `Makefile`, not that we would be adding new `Makefiles`. Adding a new `Makefile` introduces more testing and more potential sources of errors.

Also, this is technically unsupported code because it is in Pony.

@dipinhora what needs to be done to make sure that this works correctly?

Reply:
@aturley See my comment above yours. This is a standard ponyc build so the plain/simple makefile without those extra lines I mentioned should get it working correctly.